src/cli/domain/package-server-script/index.js

```diff
@@ -2,10 +2,13 @@
 
 var fs = require('fs-extra');
 var path = require('path');
-var hashBuilder = require('../../../utils/hash-builder');
+
 var bundle = require('./bundle');
+var hashBuilder = require('../../../utils/hash-builder');
+
+var webpackDefault =  { stats: 'errors-only' };
 
-var webpackDefaults =  {
+var webpackVerbose =  {
   stats: {
```

Comment:
Now that verbosity is hidden behind a flag, It might be better to rely on some more verbose [presets that webpack](https://github.com/webpack/docs/wiki/node.js-api#statstojsonoptions) offers.
What about `stats: "verbose"` or`stats: "normal"`?